app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -76,7 +75,7 @@ fun MessageItem(
                 .fillMaxWidth()
                 .let {
                     if (!message.isDeleted) it.combinedClickable(
-                        //TODO: implement some action onClick
+                        // TODO: implement some action onClick
                         onClick = { },
                         onLongClick = { onLongClicked(message) }
```

Comment:
Ping message types shouldn't be long clicked, right? Should we block it here or in the place where we open the message options bottom sheet? If the second solution then we should also remove the ripple when long clicking

Reply:
For now Ping is a System Message so long click is not implemented for it